boards/arm/stm32f072b_disco/doc/stm32f072b_disco.rst

```diff
@@ -0,0 +1,216 @@
+.. _stm32f072b_disco_board:
+
+ST STM32F072B-DISCO
+###################
+
+Overview
+********
+
+The STM32F072B-DISCO Discovery kit features an ARM Cortex-M0 based STM32F072RB
+MCU with everything required for beginners and experienced users to get
+started quickly. Here are some highlights of the STM32F072B-DISCO board:
+
+- STM32 microcontroller in LQFP64 package
+- Extension header for LQFP64 I/Os for a quick connection to the prototyping
+  board and easy probing
+- On-board ST-LINK/V2, debugger/programmer with SWD connector
+- Board power supply: through USB bus or from an external 5 V supply voltage
+- External application power supply: 3 V and 5 V
+
+- Six LEDs:
+
+       - LD1 (red/green) for USB communication
```

Comment:
This sublist looks better when it's not indented so much, 4 spaces is sufficient.  Otherwise the html that's generated has a vertical bar added on the left because it looks like a quoted text block.

Reply:
done

Reply:
actually I had to go down to 2 Spaces to remove the quote-bar.